src/features/interactions.js

```diff
@@ -29,11 +36,21 @@ export function onMouseDown(event, ViewerDOM, tool, value, props, coords = null)
   switch (tool) {
     case TOOL_ZOOM_OUT:
       let SVGPoint = getSVGPoint(value, x, y);
-      nextValue = zoom(value, SVGPoint.x, SVGPoint.y, 1 / props.scaleFactor);
+      if (props.scaleFactorMin && lessThanScaleFactorMin(props, nextValue)) {
+        console.error('Cannot zoom out past minimum scale factor of: ', props.scaleFactorMin);
```

Comment:
These errors would only show if the scaleFacorMin or scaleFactorMax were provided, see the check for `props.scaleFactorMin`. Can still remove if desired.